App.js

```diff
@@ -15,6 +15,11 @@ import gpsStrategy from './app/gps';
 import BackgroundTaskService from './app/services/BackgroundTaskService';
 import { isGPS } from './app/COVIDSafePathsConfig';
 
+//This is a hack because can't find the reason that Config.TRACING_STRATEGY is underined here when running android-bt
+if (typeof Config.TRACING_STRATEGY === 'undefined'){
```

Comment:
This is something that we probably dont want in the production code.  Please continue to use this locally to unblock any work you are doing, but the real solution is to identify why react-native-config is not working on your local development instance.